packages/grafana-data/src/transformations/matchers/valueMatchers/substringMatchers.ts

```diff
@@ -28,7 +30,13 @@ const isNotSubstringValueMatcher: ValueMatcherInfo<BasicValueMatcherOptions> = {
   get: (options) => {
     return (valueIndex: number, field: Field) => {
       const value = field.values[valueIndex];
-      return typeof value === 'string' && options.value !== '' && !value.includes(options.value);
+      return (
+        typeof value === 'string' &&
+        options.value &&
+        value &&
```

Comment:
is this the `'0'` coercion problem back?

Reply:
possibly, let me check